AP table update for components

woodbe opened the pull request:
This is to update the component listing for when mold and cast are equal in the calculation.

This is related to #16.

@n-kai will further update the table for the calculations

Commit:
AP table update for components

This is to update the component listing for when mold and cast are equal in the calculation.

This is related to #16.

@n-kai will further update the table for the calculations

n-kai committed:
Update 01-01-Fingerprint_attack.adoc

n-kai committed:
Update 01-01-Fingerprint_attack.adoc

n-kai commented:
I think it's better to remove the component column from the table (See my PR #48)

Commit:
table links

adding table links to the text

Commit:
updates based on comments

This is the update based on the feedback from @gregott to the description paragraphs.

Commit:
Propogation to other attacks

This is a full propogation of the new AP tables to all the attacks.

Commit:
Merge pull request #48 from biometricITC/AP-table-update-2

Ap table update 2
